src/test/java/org/folio/inventory/validation/experimental/TargetItemStatusValidatorTest.java

```diff
@@ -0,0 +1,93 @@
+package org.folio.inventory.validation.experimental;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import lombok.SneakyThrows;
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import org.folio.inventory.domain.items.Status;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.inventory.domain.items.ItemStatusName.IN_PROCESS;
+import static org.folio.inventory.domain.items.ItemStatusName.forName;
+
+@RunWith(JUnitParamsRunner.class)
+public class TargetItemStatusValidatorTest {
```

Comment:
I think it might make sense to have the tests split by target status in order to keep the size of the test classes small (and keep them related to the classes actually doing the work)

```suggestion
public class InProcessTargetItemStatusValidatorTest {
```